docs/source/apis/product.rst

```diff
@@ -0,0 +1,402 @@
+  
+   `GetLowestPricedOffersForSKU <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getlowestpricedoffersforsku>`_ = `get_lowest_priced_offers_for_sku() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L138>`_
+  
+   `GetLowestPricedOffersForASIN <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getlowestpricedoffersforasin>`_ = `get_lowest_priced_offers_for_asin() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L161>`_
+  
+   `GetMyFeesEstimate <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getmyfeesestimate>`_ = `get_my_fees_estimate() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L181>`_
+  
+   `GetMyPriceForSKU <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getmypriceforsku>`_ = `get_my_price_for_sku() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L196>`_
+  
+   `GetMyPriceForASIN <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getmypriceforasin>`_ = `get_my_price_for_asin() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L210>`_
+  
+   `GetProductCategoriesForSKU <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getproductcategoriesforsku>`_ = `get_product_categories_for_sku() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L224>`_
+  
+   `GetProductCategoriesForASIN <https://github.com/Ryan-Daly/python-amazon-mws/blob/develop/docs/source/apis/product.rst#getproductcategoriesforasin>`_ = `get_product_categories_for_asin() <https://github.com/python-amazon-mws/python-amazon-mws/blob/bba6c5ed2080e5864840098ecb9f6ab7f9ac2def/mws/apis/products.py#L236>`_
+  
+  
+***************
+Getting Started
+***************
+  
+Please read through gettingStarted.rst, as these docs build upon those.
+
+.. code-block:: python
+
+      import mws
+
+      access_key = “XXXXXXXXXXXXXXXXXXXX”
+      secret_key = “XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX”
+      seller_id = “XXXXXXXXXXXXXX”
+      region = “XX”
+      my_marketplace = Marketplace.US.marketplace_id
+      
+      products_api = mws.Products(access_key, secret_key, seller_id, region)
+  
+  
+  
+************************
+`ListMatchingProducts <https://docs.developer.amazonservices.com/en_US/products/Products_ListMatchingProducts.html>`_
```

Comment:
nice - there is lots of useful info in the amazon docs that have a direct impact on the use of this module.